ui/pages/[group]/[round]/index.tsx

```diff
@@ -131,22 +141,163 @@ const Page = ({
+          />
+        ),
+      },
+      {
+        Header: "Own Funding",
+        accessor: "internalFunding",
+        Cell: ({ cell }) => (
+          <FormattedNumber
+            value={cell.value / 100}
+            style="currency"
+            currencyDisplay={"symbol"}
+            currency={round?.currency}
+          />
+        ),
+      },
+      {
+        Header: "Contributions",
+        accessor: "externalFunding",
+        Cell: ({ cell }) => (
+          <FormattedNumber
+            value={cell.value / 100}
+            style="currency"
+            currencyDisplay={"symbol"}
+            currency={round?.currency}
+          />
+        ),
+      },
+      {
+        Header: "Total funding",
+        accessor: "totalFunding",
+        Cell: ({ cell }) => (
+          <FormattedNumber
+            value={cell.value / 100}
+            style="currency"
+            currencyDisplay={"symbol"}
+            currency={round?.currency}
+          />
+        ),
+      },
+      {
+        Header: "Funders",
+        accessor: "fundersCount"
+      },
+      {
+        Header: "Funding Progress",
+        accessor: "progress",
+        Cell: ({ cell }) => cell.value + "%",
+      },
+      {
+        Header: "Approvals",
+        accessor: "goodFlagCount"
+      },
+      {
+        Header: "Flags raised",
+        accessor: "raiseFlagCount"
+      }
+    ];
+
+    if (currentUser) {
+      cols.splice(round?.allowStretchGoals ? 3 : 2, 0, {
+        Header: "Your Contribution",
+        accessor: "myFunding",
+        Cell: ({ cell }) => (
+          <FormattedNumber
+            value={cell.value / 100}
+            style="currency"
+            currencyDisplay={"symbol"}
+            currency={round?.currency}
+          />
+        ),
+      })
+    }
+
+    if (round?.allowStretchGoals) {
+      cols.splice(5, 0, {
+        Header: "Stretch Goal Progress",
+        accessor: "stretchGoalProgress",
+        Cell: ({ cell }) => cell.value + "%",
+      });
+      cols.splice(1, 0, {
+        Header: "Stretch Goal",
+        accessor: "stretchGoal",
+        Cell: ({ cell }) => (
+          <FormattedNumber
+            value={cell.value / 100}
+            style="currency"
+            currencyDisplay={"symbol"}
+            currency={round?.currency}
+          />
+        ),
+      })
+    }
+
+    return cols;
+  }, [round?.currency, round?.allowStretchGoals, currentUser]);
+
   if (error) {
     console.error(error);
   }
 
   return (
     <>
-      {buckets.map((bucket) => (
-        <Link
-          href={`/${round.group?.slug ?? "c"}/${round.slug}/${bucket.id}`}
-          key={bucket.id}
-        >
-          <a className="flex focus:outline-none focus:ring rounded-lg">
-            <BucketCard bucket={bucket} round={round} />
-          </a>
-        </Link>
-      ))}
+      {typeof window !== "undefined" &&
+        window.location.hash !== "#table" &&
+        buckets.map((bucket) => (
+          <Link
```

Comment:
Maybe use the `bucketTableReview` state variable here instead

Reply:
@aerugo  asked to keep it hidden until it is tested. How should this state be controlled? using checkbox or a switch? and where to place this control component?

Reply:
@alinauroz control it through the useEffect that is looking for the `#table` anchor, it looked like that was already set up?